doc/duckdb.md

```diff
@@ -51,7 +52,9 @@ if not Path('my.db').is_file():
     urllib.request.urlretrieve(url, 'my.db')
 ```
 
-We'll use `sqlite_scanner` extension to load a sample SQLite databse into DuckDB:
+Please note that `duckdb` dependencies need to be installed. You can install the dependencies with `pip install duckdb duckdb-engine pyarrow` from your terminal or `!pip install duckdb duckdb-engine pyarrow` from this notebook.
+
```

Comment:
can you add the pip instructions in a dropdown, just like we did in the duck example?

~~~
```{dropdown} Required packages

pip install duckdb duckdb-engine pyarrow
```
~~~

Reply:
Done!